packages/mdc-textfield/_mixins.scss

```diff
@@ -156,7 +167,7 @@
 @mixin mdc-text-field-invalid_ {
   @include mdc-text-field-bottom-line-color($mdc-text-field-error-on-light);
   @include mdc-text-field-hover-bottom-line-color($mdc-text-field-error-on-light);
-  @include mdc-text-field-focused-bottom-line-color($mdc-text-field-error-on-light);
+  @include mdc-bottom-line-color($mdc-text-field-error-on-light);
```

Comment:
You're calling both `mdc-text-field-bottom-line-color` and `mdc-bottom-line-color`...Can you change `mdc-text-field-bottom-line-color` to call `mdc-bottom-line-color` internally?

```
@mixin mdc-text-field-bottom-line-color_($color) {
  @include mdc-bottom-line-color($color);

  .mdc-text-field__input {
    @include mdc-theme-prop(border-bottom-color, $color);
  }
}
```

Reply:
No, `mdc-bottom-line-color` and `mdc-text-field-bottom-line-color` are two different colors. The latter is referring to the idle state of the input `bottom-border-color` property. 

We should probably change `mdc-text-field-bottom-line-color` to be more along the lines of `mdc-text-field-bottom-border-color` to help differentiate between the two.